csync/src/csync_private.h

```diff
@@ -207,7 +207,7 @@ __attribute__ ((packed))
 #endif
 ;
 
-void csync_file_stat_free(csync_file_stat_t *st);
+OCSYNC_EXPORT void csync_file_stat_free(csync_file_stat_t *st);
```

Comment:
Qt however tries to be tidy with its export, I think we can live without it unless it poses a concrete problem.